### PR TITLE
Framework location work

### PR DESCRIPTION
**Pull request by joshua-cogliati-inl**
--------
Pull Request Description
--------
##### What issue does this change request address?
https://github.com/idaholab/raven/issues/1764

##### What are the significant changes in functionality due to this change request?

Changes the raven framework directory imports and uses to the new ravenframework from framework.

----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [ ] 1. Review all computer code.
- [ ] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts).
- [ ] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/HERON/wiki/Code-Standards) for details.
- [ ] 4. Automated Tests should pass.
- [ ] 5. If significant functionality is added, there must be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large tes.
- [ ] 6. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync.
- [ ] 7. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done.
- [ ] 8. If an analytic test is changed/added, the the analytic documentation must be updated/added.
- [ ] 9. If any test used as a basis for documentation examples have been changed, the associated documentation must be reviewed and assured the text matches the example.

### CODE REVIEW

**Comment by joshua-cogliati-inl** [comment]
This is Do Not Merge until RAVEN switches to being inside the framework package. (Current RAVEN pull request is: https://github.com/idaholab/raven/pull/1748 )

**Comment by joshua-cogliati-inl** [comment]
@PaulTalbot-INL or @worseliz This is ready for review.

File: __init__.py
Comment (dylanjm) [inline __init__.py:20]: @joshua-cogliati-inl do we need to add the raven dir to the system path here so python knows about ravenframework?

**Review by dylanjm — APPROVED** [review]
Everything looks good to me. No required documentation changes or functionality. This PR is required before idaholab/HERON#140 can be merged.

File: __init__.py
Comment (joshua-cogliati-inl) [reply __init__.py:20]: Hm, considering we have to adjust the python path to find ```__init__.py``` for TEAL, we can probably just add both the RAVEN path and the TEAL path at the same time before we get to this file.

**Pull request merged**
